lua/dressing/config.lua

```diff
@@ -12,8 +12,8 @@ local default_config = {
     -- Can be 'left', 'right', or 'center'
     title_pos = "left",
 
-    -- When true, input will start in insert mode.
-    start_in_insert = true,
+    -- The initial mode when the window opens.
+    start_mode = "insert",
```

Comment:
Can we provide a list of accepted values in the comment?
```suggestion
    -- The initial mode when the window opens (insert|visual|normal)
    start_mode = "insert",
```